package.json

```diff
@@ -1,50 +1,74 @@
 {
-  "name": "@tact-lang/opcode",
-  "version": "0.0.19",
-  "main": "dist/index.js",
-  "repository": "https://github.com/tact-lang/ton-opcode.git",
-  "author": "Steve Korshakov <steve@korshakov.com>",
-  "license": "MIT",
-  "scripts": {
-    "gen": "node ./scripts/gen-codepage.js",
-    "build": "tsc --declaration",
-    "test": "jest",
-    "clean": "rm -fr dist",
-    "release": "yarn clean && yarn build && yarn test && release-it --npm.yarn1"
-  },
-  "devDependencies": {
-    "@release-it/keep-a-changelog": "^3.1.0",
-    "@scaleton/func-debug-symbols": "^0.1.4",
-    "@ton/core": "^0.56.3",
-    "@ton/crypto": "^3.2.0",
-    "@types/jest": "^29.5.0",
-    "@types/js-yaml": "^4.0.5",
-    "@types/node": "^18.15.5",
-    "jest": "^29.5.0",
-    "js-yaml": "^4.1.0",
-    "json-schema-to-typescript": "^15.0.4",
-    "release-it": "^15.5.1",
-    "ts-jest": "^29.0.5",
-    "ts-node": "^10.9.1",
-    "tvm-disassembler": "^3.0.0",
-    "typescript": "^5.0.2"
-  },
-  "peerDependencies": {
-    "@ton/core": ">=0.49.2",
-    "@ton/crypto": "^3.2.0"
-  },
-  "publishConfig": {
-    "access": "public",
-    "registry": "https://registry.npmjs.org/"
-  },
-  "release-it": {
-    "github": {
-      "release": true
+    "name": "@tact-lang/opcode",
+    "version": "0.1.0",
+    "repository": "https://github.com/tact-lang/ton-opcode.git",
+    "author": "Tact Authors",
+    "main": "dist/index.js",
+    "license": "MIT",
+    "description": "TVM BoC disassembler",
+    "scripts": {
+        "build": "tsc --declaration",
+        "fmt": "prettier --write -l --cache .",
+        "fmt:check": "prettier --check --cache .",
+        "lint": "eslint --cache .",
+        "gen-spec": "ts-node scripts/gen-codepage.ts",
+        "test": "vitest",
+        "coverage": "c8 vitest",
+        "postinstall": "husky"
     },
-    "plugins": {
-      "@release-it/keep-a-changelog": {
-        "filename": "CHANGELOG.md"
-      }
+    "devDependencies": {
+        "@scaleton/func-debug-symbols": "0.1.5",
+        "@ton-community/func-js": "^0.9.0",
+        "@types/node": "^22.13.1",
+        "c8": "^10.1.3",
+        "eslint": "^9.19.0",
+        "eslint-plugin-unused-imports": "^4.1.4",
+        "husky": "^9.1.7",
+        "json-schema-to-typescript": "^15.0.4",
+        "prettier": "3.4.2",
+        "ts-node": "^10.9.2",
+        "typescript": "^5.7.0",
+        "typescript-eslint": "^8.22.0",
+        "vitest": "^1.4.0"
+    },
+    "dependencies": {
+        "@ton/core": "^0.60.0",
+        "@ton/crypto": "^3.3.0"
+    },
+    "publishConfig": {
+        "access": "public",
+        "registry": "https://registry.npmjs.org/"
+    },
+    "release-it": {
```

Comment:
is redundant, we don't have a `release-it` dependency

Reply:
Yeah, just copied from old package.json

Reply:
then you should delete it

Reply:
Done